alma/tests/unit/lib/ShareOfCheckout/ShareOfCheckoutHelperTest.php

```diff
@@ -0,0 +1,271 @@
+                self::orders(),
+                $expectedTotalOrders
+            ]
+        ];
+    }
+
+    public function ordersTotalPaymentMethods() {
+        $expectedTotalPaymentMethods = [
+            [
+                "payment_method_name" => "alma",
+                "orders" => [
+                    [
+                        "order_count" => 2,
+                        "amount" => 15500,
+                        "currency" => "EUR"
+                    ],
+                    [
+                        "order_count" => 1,
+                        "amount" => 10000,
+                        "currency" => "USD"
+                    ]
+                ]
+            ],
+            [
+                "payment_method_name" => "paypal",
+                "orders" => [
+                    [
+                        "order_count" => 1,
+                        "amount" => 20000,
+                        "currency" => "USD"
+                    ]
+                ]
+            ]
+        ];
+
+        return [
+            'order total payment methods' => [
+                self::orders(),
+                $expectedTotalPaymentMethods
+            ]
+        ];
+    }
+
+    public function ordersGetPayload()
+    {
+        $expectedPayload = [
+            'start_time' => 1640991600,
+            'end_time' => 1641077999,
+            'orders' => [
+                [
+                    "total_order_count"=> 2,
+                    "total_amount"=> 15500,
+                    "currency"=> "EUR"
+                ],
+                [
+                    "total_order_count"=> 2,
+                    "total_amount"=> 30000,
+                    "currency"=> "USD"
+                ]
+            ],
+            'payment_methods' => [
+                [
+                    "payment_method_name" => "alma",
+                    "orders" => [
+                        [
+                            "order_count" => 2,
+                            "amount" => 15500,
+                            "currency" => "EUR"
+                        ],
+                        [
+                            "order_count" => 1,
+                            "amount" => 10000,
+                            "currency" => "USD"
+                        ]
+                    ]
+                ],
+                [
+                    "payment_method_name" => "paypal",
+                    "orders" => [
+                        [
+                            "order_count" => 1,
+                            "amount" => 20000,
+                            "currency" => "USD"
+                        ]
+                    ]
+                ]
+            ],
+        ];
+
+        return [
+            'order get payload' => [
+                self::orders(),
+                $expectedPayload
+            ]
+        ];
+    }
+
+    public function orders()
+    {
+        $order1 = Mockery::mock(Order::class);
+        $order1->id_currency = 1;
```

Comment:
Can you create const : 
const EUR_CURRENCY_CODE = 1 ;
const USD_CURRENCY_CODE = 2; 

i think it's more easy to read after.